### DISCUSSION TITLE
v2rayN 怎么使用订阅呢

### DISCUSSION THREAD

**Discussion by wsw-black** [vvbbnn00/WARP-Clash-API#87] [Q&A]

**Comment by vvbbnn00** [comment]
似乎 v2rayN 原生并不支持 Wireguard 协议：https://github.com/2dust/v2rayN/issues/3947

**Comment by WyInnovate** [comment]
好像v2rayN要用自定义配置，比如sing_box
https://github.com/2dust/v2rayN/issues/4544

**Comment by haowen2000** [comment]
确实订阅不了

**Reply by WyInnovate** [reply]
不行吗，我还没试，那电脑上再装个clash？
